common/request.go

```diff
@@ -68,24 +73,32 @@ type NormalizedRequest struct {
 	lastUpstream      atomic.Value
 	evmBlockRef       atomic.Value
 	evmBlockNumber    atomic.Value
+
+	isCompositeRequest atomic.Bool      // Indicates if this is a top-level composite request
+	compositeType      atomic.Value     // Type of composite request (e.g., "logs-split")
+	parentRequestId    atomic.Value     // ID of the parent request (for sub-requests)
```

Comment:
Is it necessary to use atomic values for these? If there's no risk of race conditions remove usage of atomic